library/src/main/java/com/google/android/exoplayer2/text/Cue.java

```diff
@@ -79,12 +80,18 @@
    */
   public final Alignment textAlignment;
   /**
+   * The cue image.
+   */
+  public final Bitmap bitmap;
+  /**
    * The position of the {@link #lineAnchor} of the cue box within the viewport in the direction
    * orthogonal to the writing direction, or {@link #DIMEN_UNSET}. When set, the interpretation of
    * the value depends on the value of {@link #lineType}.
    * <p>
    * For horizontal text and {@link #lineType} equal to {@link #LINE_TYPE_FRACTION}, this is the
    * fractional vertical position relative to the top of the viewport.
+   * <p>
+   * If {@link #bitmap} is not null then this value is used to indicate the top position
```

Comment:
There's no reason to special case this. Why not define it to be the posiiton of the lineAnchor (i.e. top, middle or bottom of the bitmap) as in the text case?

The only thing that's special about a bitmap is that lineType must be LINE_TYPE_FRACTION (LINE_TYPE_NUMBER doesn't make sense, since we don't know how tall a line is if the text is in bitmap form).